aws_lambda_powertools/utilities/batch/base.py

```diff
@@ -384,7 +384,7 @@ def _clean(self):
             raise BatchProcessingError(
                 msg=f"All records failed processing. {len(self.exceptions)} individual errors logged"
                 f"separately below.",
-                child_exceptions=self.exceptions,
+                child_exceptions=tuple(self.exceptions),
```

Comment:
As I copied/pasted from the previous SQS error implementation, it's best to fix the type rather than casting into a new type as containers are going to be different (and hence memory too). 

I'll fix it as the types felt odd but docs were more important yesterday